service/service.py

```diff
@@ -153,15 +186,17 @@ def post(self):
         # location_url = api.url_for(WishlistResource, wishlist_id=wishlist.id, _external=True)
         location_url = '%s/wishlists/%s' % (request.base_url, wishlist.id)
         return message, status.HTTP_201_CREATED, { 'Location': location_url }
-
+        
 ######################################################################
 # GET INDEX
 ######################################################################
+@app.route('/')
```

Comment:
does this work? Does the home page also show up at `/`?

Reply:
Nope, removed.